crates/ironrdp-graphics/src/rectangle_processing.rs

```diff
@@ -1481,10 +1481,10 @@ mod tests {
             rectangles: Vec::new(),
         };
```

Comment:
Fixed 2 failing tests, it looks like previous tests were written before renaming `Rectangle` to `InclusiveRectangle` and therefore both intersection and tests logic/data were incorrect